client/src/API/TransactionMethods.ts

```diff
@@ -71,15 +71,14 @@ export const deleteItem = async (itemId: string, type: TransactionType) => {
 };
 
 export const editItem = async (
+  budgetId: BudgetIdType,
+  categoryId: string,
   itemId: string,
-  data: TransactionTransferData,
-  type: TransactionType
+  data: TransactionTransferData
 ) => {
   try {
     await axios.patch(
-      `${URL}/transactions/edit${
-        type === "expense" ? "Expense" : "Income"
-      }/${itemId}`,
+      `${URL}/transactions/editTransaction/${budgetId}/${categoryId}/${itemId}`,
       {
         title: data.title,
         amount: data.amount,
```

Comment:
You also need to include the new category ID right here under amount if the user is editing the transaction to be in a different category.

So in the params: currentCategoryID
In the headers: newCategoryID

Reply:
Ah yes! Will fix this

Reply:
Fixed